containers/withFonts/withFonts.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import FontFaceObserver from 'fontfaceobserver';
+
+const fonts = [
+  {
+    fontFamily: 'Encode Sans',
+    url: 'https://fonts.googleapis.com/css?family=Encode+Sans:400,700',
+  },
+  {
+    fontFamily: 'DIN Condensed Bold',
+    url: null,
```

Comment:
I wonder how this is working? If you've got a good idea, I'd love for some comments to be left.

Lastly, can you document the new folder in `CONTRIBUTING.md`

Reply:
> I wonder how this is working? If you've got a good idea, I'd love for some comments to be left.

I am not sure what you mean? Do mean you are not sure this should work but it is? Or do you mean I need to add more documentation to how this works?

> Lastly, can you document the new folder in CONTRIBUTING.md

I can add this. We should think about automating the adding of new directories so it updates `CONTRIBUTING.md` automagically.

Reply:
Specifically confused on the distinction between getting a font from a CDN and one within our static folder. It looks to me like a `<link>` tag will be made with a null URL? Does this somehow force a findUp to get the file path for the font name?

Reply:
I love the idea of that auto-gen. Feel free to make an issue for it and tackle it in a later PR.

Reply:
> Specifically confused on the distinction between getting a font from a CDN and one within our static folder. It looks to me like a <link> tag will be made with a null URL? Does this somehow force a findUp to get the file path for the font name?

Actually, it wont because of this check right here: https://github.com/OperationCode/front-end/pull/311/files#diff-28487aeeb80d1f1e7029e9b6cbb036fdR19

So, if the `url` is `null`, this HoC assumes that the font is being loaded locally and will not create a `link` element for it.

> I love the idea of that auto-gen. Feel free to make an issue for it and tackle it in a later PR.

Will do!

Reply:
Perfect. Okay, so then just a comment above the `url: null` field saying something to the extent of the loading being handled in the `@font-face` rule at the top of the global stylesheet.